app/src/scripts/dashboard/dashboard.jobs.jsx

```diff
@@ -84,7 +84,7 @@ let Jobs = React.createClass({
             return (
                 <div className="fade-in  panel panel-default" key={job._id}>
                     <div className="panel-heading">
-                        <div className="header clearfix">
+                        <div className="header clearfix" id={job.analysis.status}>
```

Comment:
This is going to lead to conflicting IDs with multiple jobs with the same status on the page. I'd suggest changing it to use a CSS class and maybe making the selector more specific.

Reply:
I have changed this to be a wrapping div with the class(Name) instead.  As for the selector, it is already embedded to be fairly specific in css, I can make it more specific if need be. But, as of right now it only color codes analyses.